tsl/test/expected/cagg_bgw.out

```diff
@@ -72,6 +72,8 @@ SELECT * FROM _timescaledb_catalog.continuous_agg;
 -------------------+-------------------+------------------+----------------+---------------------+-------------------+--------------+--------------------+------------------+-------------------+-----------
 (0 rows)
 
+GRANT CREATE ON SCHEMA public TO :ROLE_DEFAULT_PERM_USER;
+psql:include/cagg_bgw_common.sql:72: WARNING:  no privileges were granted for "public"
```

Comment:
Will this warning be the same both before and after PG15?

Reply:
I think so, thats why i dont see this test fail on PG12/13/14.